android/tests_backend/hardware/location.py

```diff
@@ -11,6 +11,8 @@
 
 
 class LocationProbe(HardwareProbe):
+    request_background_permission = True
```

Comment:
Should this (and the analogous entries for iOS and Cocoa) `supports_background_permission`?

Reply:
:facepalm: Yes, rush job on my part, and of course it only affected the tests I can't run locally! Fixed in https://github.com/beeware/toga/pull/2999/commits/99376e0bd75aaa43e9cf0019668ec63eb41f1950